### DISCUSSION TITLE
Copy Failed

### DISCUSSION THREAD

**Discussion by calegox** [mvdkwast/obsidian-copy-as-html#46] [Q&A]
When I select the option "Copy as HTML" from the note menu, I get an error message saying, "copy failed: Cannot read properties of null (reading 'style')"

I may have made changes to my styles in the past for some reason but I can't remember how to access the styles to even begin to troubleshoot this error.  Any help is appreciated.

**Comment by mvdkwast** [comment]
Hi @calegox, I'm sorry to hear this. I tried without success to reproduce your issue. I tried modifying/deleting/enabling/disabling custom stylesheets, but no errors. 

You should be able to fix the plugin by deleting its configuration file. It is located in your vault at `.obsidian/plugins/copy-document-as-html/data.json`. If you'd be willing to copy its contents to this discussion, it might give me a clue about what went wrong.

For reference, custom stylesheets can be provided by selecting _Provided a custom stylesheet_ at the bottom of the configuration menu. This makes a textarea with the label _Custom stylesheet_ visible where you can enter custom CSS rules.

**Comment by mvdkwast** [comment]
Closing this thread as reporter didn't follow-up and I couldn't reproduce.
